Ghidra/Processors/SuperH4/data/languages/SuperH4.sinc

```diff
@@ -2274,6 +2324,34 @@ define pcodeop mac_wOp;
 }
 
 
+# Load non-boundary alignment data (32-bit)
+# pattern 0100mmmm10101001
+# text    movua.l @<REG_M>,R0
+# arch    
+:movua.l M_0t_at,r0               is OP_0=0x4 & M_0t_at & r0 & OP_4=0xA9  {
+	r0 = (*:4 ( M_0t_at ));
+	
+	if (M_0t_at != 0) goto <increment>;
+	goto <skip>;
+	<increment>
+	M_0t_at = M_0t_at + 4;
+	<skip>
+}
+
+# Load non-boundary alignment data (32-bit)
+# pattern 0100mmmm11101001
+# text    movua.l @<REG_M>+,R0
+# arch    
+:movua.l M_0t_at,r0               is OP_0=0x4 & M_0t_at & r0 & OP_4=0xE9 {
```

Comment:
what is supposed to be the difference in sleigh here between the other `movua.l`?